site/tests/app/libraries/TokenManagerTester.php

```diff
@@ -104,4 +104,96 @@ public function testMissingApiKey() {
         $this->expectExceptionMessage('Missing claims in api token');
         TokenManager::parseApiToken($token);
     }
+
+    public function testCreateWebsocketToken() {
+        $current_time = time();
+
+        // Ensure the issued time is within 1 second from now to account for clock skew
+        $min_issued_time = $current_time - 1;
+        $max_issued_time = $current_time + 1;
+
+        // Ensure the expiration time is 5 minutes +/- 1 second from now to account for clock skew
+        $min_expired_time = $current_time + 5 * 60 - 1;
+        $max_expired_time = $current_time + 5 * 60 + 1;
+
+        $authorized_page = 'f25-sample-defaults';
+        $token = TokenManager::generateWebSocketToken('test_user', $authorized_page);
+
+        $this->assertEquals('test_user', $token->claims()->get('sub'));
+        $this->assertEquals('https://submitty.org', $token->claims()->get('iss'));
+        $this->assertCount(1, $token->claims()->get('authorized_pages'));
+        $this->assertEquals($authorized_page, array_keys($token->claims()->get('authorized_pages'))[0]);
+
+        $this->assertTrue($token->claims()->has('iat'));
+        $this->assertGreaterThanOrEqual($min_issued_time, $token->claims()->get('iat')->getTimestamp());
+        $this->assertLessThanOrEqual($max_issued_time, $token->claims()->get('iat')->getTimestamp());
+
+        $this->assertTrue($token->claims()->has('expire_time'));
+        $this->assertGreaterThanOrEqual($min_expired_time, $token->claims()->get('expire_time'));
+        $this->assertLessThanOrEqual($max_expired_time, $token->claims()->get('expire_time'));
+    }
+
+    public function testParseWebsocketToken() {
+        $future_time = time() + 5 * 60;
+        $expired_time = time() - 5 * 60;
+
+        $authorized_page = 'f25-tutorial-defaults';
+        $existing_authorized_pages = [
+            'f25-sample-defaults' => $future_time,
+            'f25-sample-chatrooms-1' => $future_time,
+            'f25-sample-chatrooms-2' => $expired_time
+        ];
+        $token = TokenManager::generateWebSocketToken('test_user', $authorized_page, $existing_authorized_pages);
+        $parsed_token = TokenManager::parseWebSocketToken($token->toString());
+
+        $this->assertEquals('test_user', $parsed_token->claims()->get('sub'));
+        $this->assertEquals('https://submitty.org', $parsed_token->claims()->get('iss'));
+        $this->assertCount(3, $parsed_token->claims()->get('authorized_pages'));
+
+        // Expired pages should be removed
+        $this->assertArrayNotHasKey('f25-sample-chatrooms-2', $parsed_token->claims()->get('authorized_pages'));
+
+        // Old pages should persist with the same expiration time
+        $this->assertEquals($future_time, $parsed_token->claims()->get('authorized_pages')['f25-sample-defaults']);
```

Comment:
you have a couple references to the old `defaults` in these tests. If you could update those to something else, that would be great